tests/styleguide.spec.ts

```diff
@@ -1,151 +1,178 @@
+import { Page, expect, test } from '@playwright/test';
+
+const isolatedSampleSectionIds: string[] = [
+  'grids-grid',
+  'navigations',
+  'icons',
+  'editors',
+];
 
 const sampleSectionIds: string[] = [
-  'sample-section-typography',
-  'sample-section-colors',
-  'sample-section-theme-color-palette',
-  'sample-section-semantic-colors',
-  'sample-section-chart-colors',
-  'sample-section-editor-colors',
-  'sample-section-grid-colors',
-  'sample-section-component-colors',
-  'sample-section-golden-layout',
-  'sample-section-buttons-regular',
-  'sample-section-buttons-outline',
-  'sample-section-buttons-inline',
-  'sample-section-buttons-socketed',
-  'sample-section-links',
-  'sample-section-progress',
-  'sample-section-inputs',
-  'sample-section-item-list-inputs',
-  'sample-section-draggable-lists',
-  'sample-section-time-slider-inputs',
-  'sample-section-dialog',
-  'sample-section-modals',
-  'sample-section-context-menus',
-  'sample-section-dropdown-menus',
-  'sample-section-navigations',
-  'sample-section-tooltips',
-  'sample-section-icons',
-  'sample-section-editors',
-  'sample-section-grids-grid',
-  'sample-section-grids-static',
-  'sample-section-grids-data-bar',
-  'sample-section-grids-quadrillion',
-  'sample-section-grids-async',
-  'sample-section-grids-tree',
-  'sample-section-grids-iris',
-  'sample-section-charts',
-  'sample-section-spectrum-buttons',
-  'sample-section-spectrum-collections',
-  'sample-section-spectrum-content',
-  'sample-section-spectrum-forms',
-  'sample-section-spectrum-overlays',
-  'sample-section-spectrum-well',
+  'typography',
+  'colors',
+  'theme-color-palette',
+  'semantic-colors',
+  'chart-colors',
+  'editor-colors',
+  'grid-colors',
+  'component-colors',
+  'golden-layout',
+  'buttons-regular',
+  'buttons-outline',
+  'buttons-inline',
+  'buttons-socketed',
+  'links',
+  'progress',
+  'inputs',
+  'item-list-inputs',
+  'draggable-lists',
+  'time-slider-inputs',
+  'dialog',
+  'modals',
+  'context-menus',
+  'dropdown-menus',
+  'tooltips',
+  'grids-static',
+  'grids-data-bar',
+  'grids-quadrillion',
+  'grids-async',
+  'grids-tree',
+  'grids-iris',
+  'charts',
+  'spectrum-buttons',
+  'spectrum-collections',
+  'spectrum-content',
+  'spectrum-forms',
+  'spectrum-overlays',
+  'spectrum-well',
 ];
 const buttonSectionIds: string[] = [
-  'sample-section-buttons-regular',
-  'sample-section-buttons-outline',
-  'sample-section-buttons-inline',
-  'sample-section-buttons-socketed',
+  'buttons-regular',
+  'buttons-outline',
+  'buttons-inline',
+  'buttons-socketed',
 ];
 
-test('UI regression test - Styleguide section count', async ({ page }) => {
-  await page.goto('/ide/styleguide');
+test.describe.configure({ mode: 'serial' });
+
+let page: Page;
+
+async function isolateAndGetSection(sectionId: string) {
+  const isolator = page.getByPlaceholder('Isolate');
+  await expect(isolator).toHaveCount(1);
+  await isolator.fill(sectionId);
```

Comment:
If you go with the "hashchange" event approach suggested in Styleguide.tsx, you would need to figure out how to set `window.location.hash = sectionId`